lib/kiwi_config.rb

```diff
@@ -159,6 +159,18 @@ def check_existance_of_extracted_files
     end
   end
 
+  def check_repositories_not_to_be_empty
+    if @system_description.repositories.empty?
+      raise(Machinery::Errors::MissingRequirement.new(
+        "The scope 'repositories' of the system description doesn't contain a" \
```

Comment:
Indent the first parameter one step more than `Machinery::Errors::MissingRequirement.new(`.